### PR TITLE
Fix a few invalid links

### PR DESCRIPTION
**Pull request by serpent5**
This fixes a few build warnings. I didn't get them all, because some of them should fall away as we drop <3.1 versions.

Internal Preview URLS:

- [Options pattern in ASP.NET Core](https://review.docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-6.0&branch=pr-en-us-24173)
- [Minimal APIs overview](https://review.docs.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis?view=aspnetcore-6.0&branch=pr-en-us-24173)
- [Migrate from ASP.NET Core 2.1 to 2.2](https://review.docs.microsoft.com/en-us/aspnet/core/migration/21-to-22?view=aspnetcore-6.0&branch=pr-en-us-24173&tabs=visual-studio)
- [Migrate from ASP.NET Core 2.2 to 3.0](https://review.docs.microsoft.com/en-us/aspnet/core/migration/22-to-30?view=aspnetcore-6.0&branch=pr-en-us-24173&tabs=visual-studio)

### CODE REVIEW

File: aspnetcore/fundamentals/configuration/options.md [hunk 1]
```diff
@@ -73,8 +73,8 @@ Using <xref:Microsoft.Extensions.Options.IOptionsSnapshot%601>, options are comp
 
 The difference between `IOptionsMonitor` and `IOptionsSnapshot` is that:
 
-* `IOptionsMonitor` is a [singleton service](xref:fundamentals/dependency-injection#singleton) that retrieves current option values at any time, which is especially useful in singleton dependencies.
-* `IOptionsSnapshot` is a [scoped service](xref:fundamentals/dependency-injection#scoped) and provides a snapshot of the options at the time the `IOptionsSnapshot<T>` object is constructed. Options snapshots are designed for use with transient and scoped dependencies.
+* `IOptionsMonitor` is a [singleton service](/dotnet/core/extensions/dependency-injection#singleton) that retrieves current option values at any time, which is especially useful in singleton dependencies.
+* `IOptionsSnapshot` is a [scoped service](/dotnet/core/extensions/dependency-injection#scoped) and provides a snapshot of the options at the time the `IOptionsSnapshot<T>` object is constructed. Options snapshots are designed for use with transient and scoped dependencies.
```
Comment (serpent5) [inline aspnetcore/fundamentals/configuration/options.md:77]:
```suggestion
* `IOptionsMonitor` is a [Singleton service](/dotnet/core/extensions/dependency-injection#singleton) that retrieves current option values at any time, which is especially useful in singleton dependencies.
* `IOptionsSnapshot` is a [Scoped service](/dotnet/core/extensions/dependency-injection#scoped) and provides a snapshot of the options at the time the `IOptionsSnapshot<T>` object is constructed. Options snapshots are designed for use with transient and scoped dependencies.
```

**Comment by serpent5** [comment]
Thanks @guardrex. I'd love to pull you in for more reviews but I know you're already overworked. 😆 🏃 😅

**Comment by guardrex** [comment]
It's kind'a the case, yes, in the short-term. My orders are for Blazor ... **Blazor** ... and **_MO BLAZOR!_** 🏃🏃🏃😄. If reviews are short, it's cool. If a PR has a lot of 🍖, then I'd probably be slowed down. This will (probably 🤞) change when we get into 22H1. I hope to get back to a more normal speed with Blazor UE passes (the *Security* node 🐘).

**Comment by Rick-Anderson** [comment]
You can always write something like
@guardrex  appreciate your feedback if you have time, full review not required

or something like that and maybe point out what you'd like him to review.

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**